libyul/backends/evm/EVMDialect.cpp

```diff
@@ -126,6 +126,9 @@ set<YulString> createReservedIdentifiers(langutil::EVMVersion _evmVersion)
 		if (!baseFeeException(instr.second))
 			reserved.emplace(name);
 	}
+	reserved.emplace("allocstk");
```

Comment:
We won't have stack addresses in Yul, only variables.
The stack is allocated and managed by LLVM.
(in case I understood this piece correctly)